src/server/controllers/user.js

```diff
@@ -15,7 +15,10 @@ const postRegisterDetails = async (req, res) => {
             data: {
                 email: email,
                 username: username,
-                password: hashedPassword
+                password: hashedPassword,
+                profile: {
+                    crate: {}
```

Comment:
Is this not supposed to be `create` instead of `crate`?

Reply:
oops! sorted

Reply:
Good spot